interpreter.go

```diff
@@ -949,8 +949,16 @@ func jsonToValue(i *interpreter, v interface{}) (value, error) {
 
 	case bool:
 		return makeValueBoolean(v), nil
-	case int, int8, int16, int32, int64:
-		return makeDoubleCheck(i, v.(float64))
+	case int:
+		return makeDoubleCheck(i, float64(v))
+	case int8:
+		return makeDoubleCheck(i, float64(v))
+	case int16:
+		return makeDoubleCheck(i, float64(v))
+	case int32:
+		return makeDoubleCheck(i, float64(v))
+	case int64:
+		return makeDoubleCheck(i, float64(v))
```

Comment:
We can't merge these `case` for type assertion.
If these `case` are merged, the type of `v` becomes `interface{}` so we can't cast the value to `float64`.